cfpb/consumerfinance.gov
# eRegs: Update workbox-build to 6.x

## Changes

- Update `workbox-build` to `6.0.2` from `^3.4.1`.


## How to test this PR

1. Pull branch, run `./frontend.sh`, and visit e.g. http://localhost:8000/rules-policy/regulations/1026/ and see that service worker works as before.
